test/Microsoft.AspNetCore.Hosting.Tests/WebHostTests.cs

```diff
@@ -67,15 +67,19 @@ public void UseStartupThrowsWithNull()
         }
```

Comment:
We weren't disposing the hosts created during tests. It shouldn't be a problem but @cesarbs suggested to make it consistent and dispose. Hence the changes.